connectors/sources/github.py

```diff
@@ -1241,10 +1242,60 @@ async def _logged_in_user(self):
         return self._user
 
     async def get_invalid_repos(self):
```

Comment:
What will be the error message for invalid repos?

My concern here is that I see there can be multiple reasons why a repo is not valid, so if we don't communicate the reason then this validation might not be as helpful

Reply:
This method will just return a list of invalid repos. The `validate_config` method will simply call this method and raise an error like `Inaccessible repositories: a, b, c`.

I agree this should be enhanced, but this PR does not intend to do so.